recipes/resiprocate/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+import tempfile
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class ResiprocateConan(ConanFile):
+    name = "resiprocate"
+    description = "The project is dedicated to maintaining a complete, correct, and commercially usable implementation of SIP and a few related protocols. "
+    topics = ("conan", "resiprocate", "sip")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.resiprocate.org"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    license = "BSD"
```

Comment:
https://github.com/resiprocate/resiprocate/blob/76508df8e06a4eb001e499bb2557853549f397e0/COPYING#L16
https://spdx.org/licenses/VSL-1.0.html

```suggestion
    license = "VSL-1.0"
```

Reply:
Thanks!